doc/releases/changelog-0.36.0.md

```diff
@@ -669,6 +677,11 @@
 * `qml.equal` can now be used with sums and products that contain operators on no wires like `I` and `GlobalPhase`.
   [(#5562)](https://github.com/PennyLaneAI/pennylane/pull/5562)
 
+* Fixed a bug that raised an error regarding expected vs acutal `dtype` when using `JAX-JIT` on a circuit that 
+  returned samples of observables containing the `qml.Identity` operator.
+  [(#5607)](https://github.com/PennyLaneAI/pennylane/pull/5607)
+
```

Comment:
Not sure if this really need to be in the changelog three different places, but it does seem to be at the very least both a breaking change and a bug fix

Reply:
I personally don't mind.